andela/vale-ah-frontend
# #164139723 Setup Testing with Enzyme and Jest

#### What does this PR do?
Sets up testing with `enzyme` and `jest`
#### Description of Task to be completed?
- install and configure `enzyme` and `jest`
- write basic test
- configure _Travis CI_
- configure _CodeClimate_ test coverage
- badge `README.md` file
#### How should this be manually tested?
- ensure that tests are passing on Travis
- verify that `README.md` has Travis and CodeClimate badges on them
#### Any background context you want to provide?
- All components should have tests: Unit tests thereafter integration and end to end tests
#### What are the relevant pivotal tracker stories?
[#164139723](https://www.pivotaltracker.com/story/show/164139723)
[#164139724](https://www.pivotaltracker.com/story/show/164139724)
[#164139728](https://www.pivotaltracker.com/story/show/164139728)
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A
